pkg/check/base.go

```diff
@@ -34,35 +35,36 @@ func (c *BaseCheck) Setup(ok string, nok string, metricName string, metricHelp s
 		Help:       fmt.Sprintf("The duration of resolver lookups %s in ms and percentiles", metricName),
 		Objectives: map[float64]float64{0.5: 0.05, 0.9: 0.01, 0.99: 0.001},
 	}, labels)
+	c.HistogramMetric = promauto.NewHistogramVec(prometheus.HistogramOpts{
+		Name: fmt.Sprintf("%s_histogram", metricName),
+		Help: fmt.Sprintf("The duration of resolver lookups %s in ms and percentiles", metricName),
```

Comment:
Since we add the new histogram let's add the initial buckets as well?  
suggestion:

// Durations in
// 2ms, 5ms, 10ms, ..., 10s, 20s
Buckets: []float64{0.002, 0.005, 0.01, 0.025, 0.05, 0.1, 0.25, 0.5, 1, 2.5, 5, 10, 20}